client/src/App.js

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
+import styled from '@emotion/styled';
+import Input from './components/Input';
+import Button from './components/Button';
+
+const Card = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 375px;
+  height: 812px;
```

Comment:
Be careful with a fixed width + height.
If you add or remove an input now, the height of the form Card is still the same.
I would simply remove the height and let it be calculated automatically.

Reply:
It's just a draft form, I will change all of this later during my frontend-week :) But thanks for the advice!